pallets/runtime/tests/src/staking/mod.rs

```diff
@@ -5410,3 +5431,45 @@ fn voting_for_pip_overlays_with_staking() {
         assert_noop!(alice_proposal(1), Error::InsufficientDeposit);
     });
 }
+
+#[test]
+fn test_with_multiple_validators_from_entity() {
+    ExtBuilder::default()
+        .validator_count(5)
+        .minimum_validator_count(5)
+        .build()
+        .execute_with(|| {
+            start_era(1);
+
+            // add new validator
+            bond_validator(50, 51, 500000);
+
+            // Add other stash and cntrl to the same did.
```

Comment:
cntrl?

Reply:
I wonder if Satyam meant centril or control.

Reply:
Probably "controller" 😜

Reply:
neh, that's too different. I am leaning towards centril. :stuck_out_tongue_winking_eye:

Reply:
It refers to the controller but I think centril is sounded better 😜